### PR TITLE
Create linters.yml

### PR DESCRIPTION
**Pull request by principles31**
Reviewing  one of my past projects (microverse media_project)

The project contains the following 

- index.html
- style.css
- readme file
- img folder

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+<iframe class="thumbnails" width="560" height="315" src="https://www.youtube.com/embed/2PW4uKpnCWI?start=1" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+<img src="download (6).png" class="channel-icon">
+<div class="information-section">
+  <h3 class="information">Is Microverse worth it? (student opinion)</h3>
+  <p class="channel-name">Microverse</p>
+</div>
+  </div>
+  </div>
```
Comment (principles31) [inline index.html:102]: I think you should follow the semantic Html Tags to make your code easier to maintain and access.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+      width: 40px;
+      height: 40px;
+      margin-top: 350px;
+  
+      border: 2px solid black;
+      border-radius:50%;
+  
+  
+  } 
+
```
Comment (principles31) [inline style.css:229]: Please apply CSS flex
 **(i)* applying CSS flex is an easy way to make the project look more attractive for your future employers.

File: index.html (hunk 1 above)
Comment (akucintavalent) [reply index.html:102]: Please provide an important and easy-to-understand reason for the suggested change. Even though my English is almost advanced I don't understand what you mean by "more comprehensive and informative". You can google "why use semantic HTML" and copy-paste a sentence or two about why to use it.

File: style.css (hunk 2 above)
Comment (akucintavalent) [reply style.css:229]: Kindly explain what you mean by "best practices". What exactly practices should the student follow? The reason you provided for the requested change ("to make your code perform correctly as required") is too vague, not to mention it is obvious. Please provide an important and easy-to-understand reason for applying the requested practices.

**Review by akucintavalent — CHANGES_REQUESTED** [review]
Hi there, @principles31 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### To highlight :tada: :champagne: 
- Your comments are polite and friendly :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AmaduKamara — COMMENTED** [review]
Hi @principles31 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is failing Linter Tests. Please make sure your linter tests passed before requesting a review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

**Comment by principles31** [comment]
@AmaduKamara Per the project requirement am supposed to review the project and provide comments not to fix the error so you need to help me how to review it.

**Comment by AmaduKamara** [comment]
Hey, @principles31, no worries if that's not in your project requirement. 
- I suggest you please implement all the changes the previous reviewers wrote about giving the student reasons as to why you asked for certain changes in your inline comment. 
- After you edit your inline comments, you can go ahead and ask for a review. I'm sure your project will be approved. 👍

- Also, Kindly do not set up a linter on a project that you're not asked to use Linters on.  For this case, you should have only copied the previous project code into this repo then work on the reviews.

**Comment by principles31** [comment]
@AmaduKamara  to the best of my knowledge I have provided those reasons, please kindly check for me if they're correct.

**Comment by akucintavalent** [comment]
@principles31, in the second comment you can mention that applying `CSS flex` is an easy way to make the project look more attractive for your future employers. 

In the first comment you can copy-paste some the answer from google:
> Semantic HTML refers to syntax that makes the HTML more comprehensible by better defining the different sections and layout of web pages. It makes web pages more informative and adaptable, allowing browsers and search engines to better interpret content.

**Comment by principles31** [comment]
@akucintavalent Thank you for the feedback.

**Review by aimemalaika — COMMENTED** [review]
Hi @principles31, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

133591850-fdcb7dd3-dd7a-4274-81e7-0e59434148f7

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
@aimemalaika Thank you, you did a great job ...keep it up

**Pull request merged**